octree/include/pcl/octree/octree_pointcloud_adjacency.h

```diff
@@ -247,9 +247,8 @@ namespace pcl
 
 }
 
-//#ifdef PCL_NO_PRECOMPILE
+// Note: Don't precompile this octree type to speed up compilation. It's probably rarely used.
```

Comment:
I think the comment should rather says something like:
> Do not precompile this octree type because it is typically used with custom leaf containers.